### PR TITLE
Request validation express

### PR DESCRIPTION
**Pull request by joliveros**

### CODE REVIEW

File: guides/request-validation-express/example-2/example-2.3.x.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// You can find your Twilio Auth Token here: https://www.twilio.com/console
+// Which can be set at runtime as follows:
+// $ TWILIO_AUTH_TOKEN=XXXXXXXXXXXXXXXXXXX node index.js
+// Please note that this will not work unless you set the TWILIO_AUTH_TOKEN
+// environment constiable.
```
Comment (juancarlospaco) [inline guides/request-validation-express/example-2/example-2.3.x.js]: Whats an "environment constiable" :grey_question:

Comment (joliveros) [inline guides/request-validation-express/example-2/example-2.3.x.js]: Ouch! nice catch @juancarlospaco!

**Comment by juancarlospaco** [comment]
Looks good !. :+1:

**Comment by acamino** [comment]
LGTM. :shipit:

**Pull request merged**